modules/javafx.base/src/test/java/test/javafx/util/converter/LocalDateTimeStringConverterTest.java

```diff
@@ -104,6 +104,15 @@ public LocalDateTimeStringConverterTest(LocalDateTimeStringConverterVariant conv
         this.locale = null;
         this.formatter = null;
         this.parser = null;
+
+        final var version = Runtime.Version.parse(System.getProperty("java.version"));
+        if (version.major() < 20) {
+            // TODO: This can be removed when the minimum version of boot jdk
+            // for JFX build is updated to JDK20 or above.
+            JAPANESE_DATE_STRING = "Saturday, January 12, 60 Shōwa, 12:34:56 PM";
+        } else {
+            JAPANESE_DATE_STRING = "Saturday, January 12, 60 Shōwa, 12:34:56\u202fPM";
+        }
```

Comment:
Minor: I think this could move to the (static) `@BeforeClass` method rather than being recomputed for each instance.

Reply:
It is now moved to the `@BeforeClass` method.